### PR TITLE
Adding 2018-06-01 schemas for Compute

### PR DESCRIPTION
**Pull request by waynekuo**

### CODE REVIEW

File: schemas/2018-06-01/Microsoft.Compute.Extensions.json [hunk 1]
```diff
@@ -0,0 +1,2025 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Compute.Extensions.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Compute.Extensions",
+  "description": "Microsoft Compute Extensions Resource Types",
+  "definitions": {
```
Comment (TianoMS) [inline schemas/2018-06-01/Microsoft.Compute.Extensions.json]: >definitions [](start = 3, length = 11)

can we still put an empty "resourceDefinitions" property here?

File: tests/2018-06-01/Microsoft.Compute.tests.json [hunk 2]
```diff
@@ -0,0 +1,1407 @@
+{
+  "tests": [
```
Comment (TianoMS) [inline tests/2018-06-01/Microsoft.Compute.tests.json]: >tests [](start = 3, length = 5)

we have another set of tests in /tools/templateFiles, which are basically some test templates. Can we add the test resources to TestTemplate.json and TestTemplate.preview.json? 

Should be just a copy/paste task.

**Comment by Tiano2017** [comment]
Looks good overall. I'll need to run some service CITs to make sure nothing breaks.

File: tests/2018-06-01/Microsoft.Compute.tests.json (hunk 2 above)
Comment (waynekuo) [reply tests/2018-06-01/Microsoft.Compute.tests.json]: Sorry, can you explain this comment a bit more? Do you want me to copy everything in this test file to TestTemplate.json and TestTemplate.preview.json?

**Pull request merged**